src/components/NoteDetails.js

```diff
@@ -0,0 +1,193 @@
+  //   frequency: 92.49861,
+  //   string: "C"
+  // },
+  '23': {
+    noteName: "G",
+    noteNameOctave: "C2",
+    isAccidental: false,
+    frequency: 97.99886,
+    string: "G"
+  },
+  // '24': {
+  //   noteName: "A-Flat", // can also be G-Sharp
+  //   noteNameOctave: "G#2",
+  //   isAccidental: true,
+  //   frequency: 103.8262,
+  //   string: "G"
+  // },
+  '25': {
+    noteName: "A",
+    noteNameOctave: "A2",
+    isAccidental: false,
+    frequency: 110,
+    string: "G"
+  },
+  '26': {
+    noteName: "B-Flat", // can also be A-Sharp
+    noteNameOctave: "A#2",
+    isAccidental: true,
+    frequency: 116.5409,
+    string: "G"
+  },
+  '27': {
+    noteName: "B",
+    noteNameOctave: "B2",
+    isAccidental: false,
+    frequency: 123.4708,
+    string: "G"
+  },
+  '28': {
+    noteName: "C",
+    noteNameOctave: "C3",
+    isAccidental: false,
+    frequency: 130.8128,
+    string: "G"
+  },
+  // '29': {
+  //   noteName: "C-Sharp",
+  //   noteNameOctave: "C#3",
+  //   isAccidental: true,
+  //   frequency: 138.5913,
+  //   string: "G"
+  // },
+  '30': {
+    noteName: "D",
+    noteNameOctave: "D3",
+    isAccidental: false,
+    frequency: 146.8324,
+    string: "D"
+  },
+  // '31': {
+  //   noteName: "E-Flat", // can also be D-Sharp
+  //   noteNameOctave: "D#3",
+  //   isAccidental: true,
+  //   frequency: 155.5635,
+  //   string: "D"
+  // },
+  '32': {
+    noteName: "E",
+    noteNameOctave: "E3",
+    isAccidental: false,
+    frequency: 164.8138,
+    string: "D"
+  },
+  '33': {
+    noteName: "F",
+    noteNameOctave: "F3",
+    isAccidental: false,
+    frequency: 174.6141,
+    string: "D"
+  },
+  '34': {
+    noteName: "F-Sharp",
+    noteNameOctave: "F#3",
+    isAccidental: true,
+    frequency: 184.9972,
+    string: "D"
+  },
+  '35': {
+    noteName: "G",
+    noteNameOctave: "G3",
+    isAccidental: false,
+    frequency: 195.9977,
+    string: "D"
+  },
+  // '36': {
+  //   noteName: "G-Sharp", // can also be A-Flat
+  //   noteNameOctave: "G#3",
+  //   isAccidental: true,
+  //   frequency: 207.6523,
+  //   string: "D"
+  // },
+  '37': {
+    noteName: "A",
+    noteNameOctave: "A3",
+    isAccidental: false,
+    frequency: 220,
+    string: "A"
+  },
+  // '38': {
+  //   noteName: "A-Sharp",
+  //   noteNameOctave: "A#3",
+  //   isAccidental: true,
+  //   frequency: 233.0819,
+  //   string: "A"
+  // },
+  '39': {
+    noteName: "B",
+    noteNameOctave: "B3",
+    isAccidental: false,
+    frequency: 246.9417,
+    string: "A"
+  },
+  '40': {
+    noteName: "C",
+    noteNameOctave: "C4", // middle C
+    isAccidental: false,
+    frequency: 261.6256,
+    string: "A"
+  },
+  '41': {
+    noteName: "C-Sharp",
+    noteNameOctave: "C#4",
+    isAccidental: true,
+    frequency: 277.1826,
+    string: "A"
+  },
+  '42': {
+    noteName: "D",
+    noteNameOctave: "D4",
+    isAccidental: false,
+    frequency: 293.6648,
+    string: "A"
+  }
+}
+
+export default Notes;
```

Comment:
with these constant values, is better to export a named module `export const Notes = {... `

Reply:
Cool, I want to ask you about this tomorrow!